### ISSUE TITLE
Double Backspace in Empty First Paragraph Triggers Back in Firefox

### ISSUE THREAD

**Issue by mrwweb** [WordPress/gutenberg#6961]
**Describe the bug**
In Windows 10 on the latest Firefox, hitting backspace twice in quick succession will make the browser try to go back one page when the cursor is in an empty paragraph that is the first block of the page.

In Chrome, the cursor disappears and it's unclear where focus goes exactly.

**To Reproduce**
Steps to reproduce the behavior (using Firefox):
1. Make a new page
2. Get active cursor in the first paragraph block
3. Hit Backspace twice quickly
4. Browser triggers alert: "This page is asking you to confirm that you want to leave - data you have entered may not be saved."

_Note: While Chrome doesn't display this dangerous behavior that could result in lost work, it doesn't meet my expected behavior either._

**Expected behavior**
I can see multiple expected bevhaviors, but I would prefer nothing to happen. My cursor is at the start of the document so backspace has nowhere to go. The cursor should stay in place and I can resume typing immediately.

Relatedly, if the cursor is in the first block and user presses LEFT ARROW, it goes to the title rather than staying "in the document". Pressing DOWN, RIGHT, or DEL at the end of the document does not move the cursor/focus as expected.

**Desktop (please complete the following information):**
 - OS: Win10
 - Browser: Firefox
 - Version 60.0.1 (seen on earlier versions as well

**Smartphone (please complete the following information):**
N/A

**Additional context**
- Gutenberg Version: 2.9.2

**Comment by danielbachhuber** [comment]
From #7321:

> I'd expect to remain focused within the default paragraph block if I use "Delete" within an empty Gutenberg editor. The Classic Editor exhibits this behavior.

**Comment by anevins12** [comment]
I need someone to try to replicate this. I tried this in Browserstack using Windows 10, FF60, cannot replicate

**Comment by danielbachhuber** [comment]
I can still replicate this in FF 61.0.2 (64-bit) on a Mac. #7321 has a GIF of what I've experienced in the past.

**Comment by anevins12** [comment]
I see, it would be interesting to know why the focus leaves the input field on backspace

**Comment by mrwweb** [comment]
Reconfirmed also on Firefox 61.0.2 (64-bit), Win 10, Gberg 3.6.2.

@anevins12 You're right that the first backspace removes focus and it moves back to the body element.

**Comment by iseulde** [comment]
Wouldn't be surprised if this would be fixed by #11287.

**Comment by danielbachhuber** [comment]
I've tested again and confirmed I've confirmed that this issue is fixed in FF 63.0 on Mac with 4.2.0-alpha-a696e50 (nightly build as of a696e50)
